akka-docs/src/main/paradox/typed/persistence-snapshot.md

```diff
@@ -121,6 +121,14 @@ Deleting events in Event Sourcing based applications is typically either not use
 By deleting events you will lose the history of how the system changed before it reached current state, which is
 one of the main reasons for using Event Sourcing in the first place.
 
+If snapshots are triggered using the predicate-based api (@scala[`snapshotWhen`]@java[`shouldSnapshot`]), events are deleted once a snapshot is created. Events are getting deleted up until the last snapshot unless snapshot-retention is enabled. It can be enabled as follows:
```

Comment:
I think that can be misinterpreted. By default events are not deleted when using the predicate based snapshot api, but event deletion can be enabled by... Scala deleteEventsOnSnapshot with deleteEventsOnSnapshot parameter ... Java deleteEventsOnSnapshot